src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/inScalar.txt

```diff
@@ -34,21 +34,19 @@ true
 >> "" in ""
 true
 
-
-// If Blank is on either side, operator is false. 
-// Use If() to coerce Blank to string. 
->> Blank() in Blank()
```

Comment:
From the other comment - `Blank() in Blank()` is ambiguous, since the `in` operator is overloaded. This is also not something that one would write in a real app. I'm replacing this test with others where we have "typed blank values"